### ISSUE TITLE
Wrong documentation for `knn_test` and `knn_trn`

### ISSUE THREAD

**Issue by blancadomene** [CasedUgr/KnowSeq#59]
According to the documentation, the `knn_test` function should return a list with four items:
https://github.com/CasedUgr/KnowSeq/blob/master/R/knn_test.R#L11
```R 
#' @return A list that contains four objects. The confusion matrix, the accuracy, the sensitibity and the specificity for each genes.
```

Whereas the implementation actually return a list with six items:
https://github.com/CasedUgr/KnowSeq/blob/master/R/knn_test.R#L150-L152
```R 
results <- list(cfMatList,accVector,sensVector,specVector,f1Vector,predictsVector)
names(results) <- c("cfMats","accVector","sensVector","specVector","f1Vector","predictions")
invisible(results)
```

Same for `knn_trn`.
https://github.com/CasedUgr/KnowSeq/blob/master/R/knn_trn.R#L10
```R 
#' @return A list that contains four objects. The confusion matrix for each fold, the accuracy, the sensitibity and the specificity for each fold and each genes, and the best k found for the knn algorithm after tuning.
```
https://github.com/CasedUgr/KnowSeq/blob/master/R/knn_trn.R#L199-L201
```R 
results_cv <- list(cfMatList,accuracyInfo,sensitivityInfo,specificityInfo,F1Info,bestK,predictions)
    names(results_cv) <- c("cfMats","accuracyInfo","sensitivityInfo","specificityInfo","F1Info","bestK",'predictions')
    invisible(results_cv)
```

According to the documentation, `knn_trn` and `knn_test` should return pretty much the same information in the same format: the confusion matrix, the accuracy, the sensitibity and the specificity.

This is no longer true, since they return different formats, hence it's more difficult to work with this library.

**Comment by pacocp** [comment]
Hi Blanca,

Thank you so much for bringing this up. You are completely right, the documentation is outdated and it must be updated. We will work this out in a new update and release, also we'll reorder the output of the two functions. 

Thanks again!

Paco

**Comment by pacocp** [comment]
Fix on pull request #58
